CITATIONS.md

```diff
@@ -40,6 +40,9 @@
 
   > Peltzer, A., Jäger, G., Herbig, A., Seitz, A., Kniep, C., Krause, J., & Nieselt, K. (2016). EAGER: efficient ancient genome reconstruction. Genome Biology, 17(1), 1–14. doi: [10.1186/s13059-016-0918-z](https://doi.org/10.1186/s13059-016-0918-z)
 
+  - MTNucRatioCalculator (Unpublished)
```

Comment:
```suggestion

  - MTNucRatioCalculator (Unpublished)
```

I've asked Alex Peltzer if he can get a Zenodo DOI for us

Reply:
But we can merge now anyway while we wait (we will have to wait untilthe weekend)